src/Helpers.js

```diff
@@ -301,15 +303,15 @@ export const ICONLINKS = {
 export const platformTokens = {
   421611: {
     // arbitrum testnet
-    TCR: {
-      name: "TCR",
-      symbol: "TCR",
+    MYC: {
+      name: "MYC",
+      symbol: "MYC",
       decimals: 18,
       address: getContract(ARBITRUM_TESTNET, "TCR"),
-      imageUrl: "https://assets.coingecko.com/coins/images/18271/small/tracer_logo.png?1631176676",
```

Comment:
I think this means that users cant addTokenToWallet or does that still work?

Reply:
negative, will revert: 

![image](https://user-images.githubusercontent.com/19278287/183008232-3e8ebfd6-2323-4a7d-a36b-61b3ab05f6f2.png)